src/main/java/com/hongliangjie/fugue/topicmodeling/LDA/LDAModel.java

```diff
@@ -0,0 +1,42 @@
+package com.hongliangjie.fugue.topicmodeling.lda;
+
+import com.hongliangjie.fugue.Message;
+import com.hongliangjie.fugue.serialization.Model;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by liangjie on 3/6/16.
+ */
+public class LDAModel extends Model{
+    private double[] alpha = new double[1];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'alpha' could be replaced by a local variable.](https://www.codacy.com/app/hongliangjie/Fugue/file/2178414323/issues/source?bid=3162994&fileBranchId=3163615#l14)